src/translations/en.ts

```diff
@@ -1,6 +1,28 @@
 import { Translations } from './Translations';
 
 export const enTranslations: Translations['en'] = {
+  onboarding: {
+    pages: {
+      0: {
+        title: 'Welcome to Karmine Corp',
+        description: 'Follow the matches of your favorite teams',
+      },
+      1: {
+        title: 'Notifications',
+        description: 'Receive notifications to not miss anything',
```

Comment:
Invite the user to select the game for which they want to receive notifications.